### ISSUE TITLE
pocket_updater keeps running when showing `--help` or `--version`

### ISSUE THREAD

**Issue by davewongillies** [mattpannella/pocket_core_autoupdate_net#15]
When running `pocket_updater` with `--help` or `--version`, it keeps running instead of exiting

```
$ ./pocket_updater --help
pocket_updater 1.0.0
Copyright (C) 2022 pocket_updater

  -u, --update    Automatically download newest core list without asking.

  -p, --path      Absolute path to install location

  -a, --all       Extract all release assets, instead of just ones containing openFPGA cores.

  --help          Display this help screen.

  --version       Display version information.

Analogue Pocket Core Updater v2.0.1
Checking for updates...
Download master cores list file from github? (This will overwrite your current file) [y/n]: (default is y)
```

```
$ ./pocket_updater --version
pocket_updater 1.0.0
Analogue Pocket Core Updater v2.0.1
Checking for updates...
Download master cores list file from github? (This will overwrite your current file) [y/n]: (default is y)
```

Also its printing two versions

**Comment by mattpannella** [comment]
thanks! i'll clean this up next time i get a chance to sit down and work on it

**Comment by mattpannella** [comment]
this should be fixed in the next release soon

**Comment by mattpannella** [comment]
resolved in 2.1.0
